src/LibVLCSharp/Shared/LibVLC.cs

```diff
@@ -286,31 +316,11 @@ public LibVLC(params string[] options)
         /// <param name="enableDebugLogs">enable verbose debug logs</param>
         /// <param name="options">list of arguments (should be NULL)</param>
         public LibVLC(bool enableDebugLogs, params string[] options)
-            : base(() => MarshalUtils.CreateWithOptions(PatchOptions(options, enableDebugLogs), Native.LibVLCNew), Native.LibVLCRelease)
-        {
-            _gcHandle = GCHandle.Alloc(this);
-        }
-
-        /// <summary>
-        /// Make dirty hacks to include necessary defaults on some platforms.
-        /// </summary>
-        /// <param name="options">The options given by the user</param>
-        /// <param name="enableDebugLogs">enable debug logs</param>
-        /// <returns>The patched options</returns>
-        static string[] PatchOptions(string[] options, bool enableDebugLogs = false)
-        {
-            if(enableDebugLogs)
-            {
-                options = options.Concat(new[] { "--verbose=2" }).ToArray();
-            }
-#if UWP
-            return options.Concat(new[] {"--aout=winstore", "--audio-resampler=speex_resampler"}).ToArray();
-#elif ANDROID
-            return options.Concat(new[] {"--audio-resampler=soxr"}).ToArray();
-#else
-            return options;
-#endif
-        }
+            : this(new LibVLCOptionsBuilder()
+                        .AddOptions(options)
+                        .EnableDebugLogs()
```

Comment:
I think the logs are enabled when running with `new LibVLC(enableDebugLogs: false)` here?